src/mode/meta.js

```diff
@@ -1,5 +1,6 @@
 import CodeMirror from "codemirror/lib/codemirror"
+import * as ABAP from "./abap/abap.js";
```

Comment:
I think we may want to skip the `import` here, its not actually being used.

`meta.js` is designed to be a manifest to allow modes to be lazily loaded/imported rather than being concatenated all together.